stjude/proteinpaint
# Hic.14

## Description
Fixes for showing the fragments in the detail view. On zoom, the fragments are clearly displayed and the info bar updates appropriately. 


To test: 
1. Load http://localhost:3000/example.hic.detail.html, using `proteinpaint_demo/hg19/hic/hic_demo.hic`, enzyme `MboI`, and the following positions 
```
position1: "chr8:128782746-128795246",
position2: "chr8:128782746-128795246"
```
2. Check from app launch - With Enzyme: http://localhost:3000/?genome=hg19&hicfile=proteinpaint_demo/hg19/hic/hic_demo.hic&enzyme=MboI
3. Check from app launch  - Without Enzyme: http://localhost:3000/?genome=hg19&hicfile=proteinpaint_demo/hg19/hic/hic_demo.hic&enzyme=MboI

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
